### ISSUE TITLE
Says Note but Pubrules rejects

### ISSUE THREAD

**Issue by michael-n-cooper** [w3c/respec#653]
The latest Respec, at time of this filing, inserts the following text into the Status:

It is expected to become a <abbr title="World Wide Web Consortium">W3C</abbr> Note.

Pubrules rejects this as and declares the entire patent policy paragraph invalid (section 10). I can't point to a URI to demonstrate this because I am manually removing the offending text in order to pass pubrules and be able to publish. The version of Pubrules is at https://www.w3.org/2005/07/pubrules?uimode=checker_full&uri= and raises this error on a doc with noRecTrack: true in the respec config.

I previously said this error had gone away after a hard refresh.

https://lists.w3.org/Archives/Public/spec-prod/2016JanMar/0062.html

That turns out to be incorrect. I must have already reverted my staging doc to the known-working version of respec when I asserted that. This issue is still present in the current version of Respec.

**Comment by halindrome** [comment]
I feel like this is a change @marcoscaceres pushed recently.  If so, can he please back it out and then test a note against pubrules?

**Comment by marcoscaceres** [comment]
I'll investigate. @michael-n-cooper, in the future (or if you are around), please paste in your ReSpec config. It helps us greatly with debugging.

**Comment by michael-n-cooper** [comment]
Here's the respec config. And BTW the behaviour is observed on 
https://rawgit.com/w3c/aria/Mar_2016_Heartbeats/practices/aria-practices.html 
though that URI will only be valid temporarily, and is not one that can 
be checked in pubrules directly. The respec config from that doc is (the 
last two lines use respec extension functions so will be unfamiliar):


var respecConfig = { // Embed RDFa data in the output. doRDFa: '1.1', 
includePermalinks: true, permalinkEdge: true, permalinkHide: false, // 
Specification status (e.g., WD, LC, NOTE, etc.). If in doubt use ED. 
specStatus: "WD", // crEnd: "2012-04-30", // perEnd: "2013-07-23", 
publishDate: "2016-03-17", noRecTrack: true, diffTool: 
"http://www.aptest.com/standards/htmldiff/htmldiff.pl", // The 
specifications short name, as in http://www.w3.org/TR/short-name/ 
shortName: "wai-aria-practices-1.1", // If you wish the publication date 
to be other than today, // set publishDate. // publishDate: 
"2009-08-06", copyrightStart: "2015", // If there is a previously 
published draft, uncomment this // and set its YYYY-MM-DD date and its 
maturity status. // previousPublishDate: "2015-11-19", previousMaturity: 
"WD", // prevRecURI: "", // previousDiffURI: "", // If there a publicly 
available Editors Draft, this is the link edDraftURI: 
"http://w3c.github.io/aria/practices/aria-practices.html", // If this is 
a LCWD, uncomment and set the end of its review period // lcEnd: 
"2012-02-21", // Editors, add as many as you like. // “name” is the only 
required field. editors: [ { name: "Matt King", mailto: "mck@fb.com", 
company: "Facebook", companyURI: "http://www.facebook.com/" }, { name: 
"James Nurthen", mailto: "james.nurthen@oracle.com", company: "Oracle 
Corporation", companyURI: "http://www.oracle.com/" }, { name: "Michael 
Cooper", url: 'http://www.w3.org/People/cooper/', mailto: 
"cooper@w3.org", company: "W3C", companyURI: "http://www.w3.org/" }, { 
name: "Michiel Bijl", mailto: "mbijl@paciellogroup.com", company: "The 
Paciello Group", companyURI: "https://www.paciellogroup.com/" }, { name: 
"Joseph Scheuhammer", company: "Inclusive Design Research Centre, OCAD 
University", companyURI: "http://idrc.ocad.ca/", note: "Previous Editor" 
}, { name: "Lisa Pappas", company: "SAS", companyURI: 
"http://www.sas.com/", note: "Previous Editor" }, { name: "Rich 
Schwerdtfeger", company: "IBM Corporation", companyURI: 
"http://ibm.com/", note: "Previous Editor" }, ], // Authors, add as many 
as you like. // This is optional, uncomment if you have authors as well 
as editors. // Same format and requirements as editors. // authors: [ // 
{ // name: "Your Name", // url: "http://example.org/", // company: "Your 
Company", // companyURI: "http://example.com/" // }, // ], // Spec URLs 
ariaSpecURLs: { "ED": "http://w3c.github.io/aria/aria/aria.html", "FPWD" 
: "http://www.w3.org/TR/wai-aria-1.1/", "WD" : 
"http://www.w3.org/TR/wai-aria-1.1/", "REC": 
"http://www.w3.org/TR/wai-aria/" }, accNameURLs: { "ED": 
"http://w3c.github.io/aria/accname-aam/accname-aam.html", "WD" : 
"http://www.w3.org/TR/accname-aam-1.1/", "FPWD": 
"http://www.w3.org/TR/accname-aam-1.1/", "REC": 
"http://www.w3.org/TR/accname-aam-1.1/" }, coreMappingURLs: { "ED": 
"http://w3c.github.io/aria/core-aam/core-aam.html", "WD" : 
"http://www.w3.org/TR/core-aam-1.1/", "FPWD": 
"http://www.w3.org/TR/core-aam-1.1/", "REC": 
"http://www.w3.org/TR/core-aam-1.1/" }, // alternateFormats: [ // { // 
uri: 'aria-practices-diff.html', // label: "Diff from Previous 
Recommendation" // }, // { // uri: 'aria-practices.ps', // label: 
"PostScript version" // }, // { // uri: 'aria-practices.pdf', // label: 
"PDF version" // } // ], // errata: 
'http://www.w3.org/2010/02/rdfa/errata.html', // name of the WG wg: 
"Accessible Rich Internet Applications Working Group", // URI of the 
public WG page wgURI: "http://www.w3.org/WAI/ARIA/", // Name (without 
the @w3c.org) of the public mailing // to which comments are due. 
wgPublicList: "public-aria", // URI of the patent status for this WG, 
for Rec-track documents // !!!! IMPORTANT !!!! // This is important for 
Rec-track documents, do not copy a patent URI // from a random document 
unless you know what you're doing. // If in doubt ask your friendly 
neighbourhood Team Contact. wgPatentURI: 
"http://www.w3.org/2004/01/pp-impl/83726/status", maxTocLevel: 4, 
localBiblio: biblio, preProcess: [ linkCrossReferences ] };

**Comment by marcoscaceres** [comment]
Beautified config: 

```JS
var respecConfig = { // Embed RDFa data in the output. doRDFa: '1.1', 
    includePermalinks: true,
    permalinkEdge: true,
    permalinkHide: false, // 
    Specification status(e.g., WD, LC, NOTE, etc.).If in doubt use ED.
    specStatus: "WD", // crEnd: "2012-04-30", // perEnd: "2013-07-23", 
    publishDate: "2016-03-17",
    noRecTrack: true,
    diffTool: "http://www.aptest.com/standards/htmldiff/htmldiff.pl", // The 
    specifications short name,
    as in http: //www.w3.org/TR/short-name/ 
        shortName: "wai-aria-practices-1.1", // If you wish the publication date 
    to be other than today, // set publishDate. // publishDate: 
    "2009-08-06",
    copyrightStart: "2015", // If there is a previously 
    published draft,
    uncomment this // and set its YYYY-MM-DD date and its 
    maturity status. // previousPublishDate: "2015-11-19", previousMaturity: 
    "WD", // prevRecURI: "", // previousDiffURI: "", // If there a publicly 
    available Editors Draft,
    this is the link edDraftURI: "http://w3c.github.io/aria/practices/aria-practices.html", // If this is 
    a LCWD,
    uncomment andset the end of its review period // lcEnd: 
        "2012-02-21", // Editors, add as many as you like. // “name” is the only 
    required field.editors: [{
        name: "Matt King",
        mailto: "mck@fb.com",
        company: "Facebook",
        companyURI: "http://www.facebook.com/"
    }, {
        name: "James Nurthen",
        mailto: "james.nurthen@oracle.com",
        company: "Oracle 
        Corporation ", companyURI: "
        http: //www.oracle.com/" }, { name: "Michael 
            Cooper ", url: 'http://www.w3.org/People/cooper/', mailto: 
        "cooper@w3.org",
        company: "W3C",
        companyURI: "http://www.w3.org/"
    }, {
        name: "Michiel Bijl",
        mailto: "mbijl@paciellogroup.com",
        company: "The 
        Paciello Group ", companyURI: "
        https: //www.paciellogroup.com/" }, { name: 
            "Joseph Scheuhammer",
        company: "Inclusive Design Research Centre, OCAD 
        University ", companyURI: "
        http: //idrc.ocad.ca/", note: "Previous Editor" 
    }, {
        name: "Lisa Pappas",
        company: "SAS",
        companyURI: "http://www.sas.com/",
        note: "Previous Editor"
    }, {
        name: "Rich 
        Schwerdtfeger ", company: "
        IBM Corporation ", companyURI: 
        "http://ibm.com/",
        note: "Previous Editor"
    }, ], // Authors, add as many 
    as you like. // This is optional, uncomment if you have authors as well 
    as editors. // Same format and requirements as editors. // authors: [ // 
    { // name: "Your Name", // url: "http://example.org/", // company: "Your 
        Company ", // companyURI: "
        http: //example.com/" // }, // ], // Spec URLs 
            ariaSpecURLs: {
                "ED": "http://w3c.github.io/aria/aria/aria.html",
                "FPWD": "http://www.w3.org/TR/wai-aria-1.1/",
                "WD": "http://www.w3.org/TR/wai-aria-1.1/",
                "REC": "http://www.w3.org/TR/wai-aria/"
            }, accNameURLs: {
                "ED": "http://w3c.github.io/aria/accname-aam/accname-aam.html",
                "WD": "http://www.w3.org/TR/accname-aam-1.1/",
                "FPWD": "http://www.w3.org/TR/accname-aam-1.1/",
                "REC": "http://www.w3.org/TR/accname-aam-1.1/"
            }, coreMappingURLs: {
                "ED": "http://w3c.github.io/aria/core-aam/core-aam.html",
                "WD": "http://www.w3.org/TR/core-aam-1.1/",
                "FPWD": "http://www.w3.org/TR/core-aam-1.1/",
                "REC": "http://www.w3.org/TR/core-aam-1.1/"
            }, // alternateFormats: [ // { // 
            uri: 'aria-practices-diff.html', // label: "Diff from Previous 
            Recommendation " // }, // { // uri: 'aria-practices.ps', // label: 
        "PostScript version" // }, // { // uri: 'aria-practices.pdf', // label: 
        "PDF version" // } // ], // errata: 
        'http://www.w3.org/2010/02/rdfa/errata.html', // name of the WG wg: 
        "Accessible Rich Internet Applications Working Group", // URI of the 
        public WG page wgURI: "http://www.w3.org/WAI/ARIA/", // Name (without 
            the @w3c.org) of the public mailing // to which comments are due. 
    wgPublicList: "public-aria", // URI of the patent status for this WG, 
    for Rec - track documents // !!!! IMPORTANT !!!! // This is important for 
    Rec - track documents,
    do not copy a patent URI // from a random document 
    unless you know what you 're doing. // If in doubt ask your friendly 
    neighbourhood Team Contact.wgPatentURI: "http://www.w3.org/2004/01/pp-impl/83726/status",
    maxTocLevel: 4,
    localBiblio: biblio,
    preProcess: [linkCrossReferences]
};

```

**Comment by michael-n-cooper** [comment]
The change does not fix the problem with Pubrules rejecting the generated Status paragraph. It just moves it to a different place that Pubrules does not accept. We could argue about whether Pubrules has reasonable expectations, but I just need a clean Pubrules report in order to publish.

**Comment by halindrome** [comment]
Uggh.  I am feeling stabby now.  @marcoscaceres I am ripping this crap out until we get pubrules fixed.

**Comment by marcoscaceres** [comment]
Strange. I ran the document over pub rules yesterday and it seemed happy.

**Comment by halindrome** [comment]
New pubrules or old?  Webmaster only tests against old :(

**Comment by marcoscaceres** [comment]
Wait, what?

**Comment by halindrome** [comment]
Echidna uses the new pubrules checker magic, but the webmaster does not.  So we need to be testing our output against the old pubrules checker.  That's the standard.  Supposedly changing in July, right @plehegar ?

**Comment by marcoscaceres** [comment]
I think this was fixed by #661.

**Comment by plehegar** [comment]
hu? the webmaster uses the new pubrules. The old pubrules has been retired.
